vsintegration/src/FSharp.Editor/DocComments/XMLDocumentation.fs

```diff
@@ -210,13 +213,34 @@ module internal XmlDocumentation =
             with _ ->
                 None
 
-        member _.CollectSummary(collector: ITaggedTextCollector) =
+        member _.CollectSummary(collector: ITaggedTextCollector, showRemarks) =
             match Seq.tryHead (doc.Descendants(XName.op_Implicit "summary")) with
             | None -> ()
             | Some el ->
                 EnsureHardLine collector
                 WriteElement collector el
 
+            match Seq.tryHead (doc.Descendants(XName.op_Implicit "returns")) with
+            | None -> ()
+            | Some el ->
+                AppendHardLine collector
+                AppendHardLine collector
+                AppendOnNewLine collector (SR.ReturnsHeader())
+                AppendHardLine collector
+                collector.Add(tagSpace "    ")
+                WriteElement collector el
+
+            if showRemarks then
+                match Seq.tryHead (doc.Descendants(XName.op_Implicit "remarks")) with
```

Comment:
Thanks for adding this setting!

Sorry to flip-flop, but given what you pointed out about how it is for C#/VB, I guess it probably does make more sense to have "remarks" before "return," like you originally had it.

Reply:
It may be better to put the returns before remarks, as the returns is more important, and lines number in the summary section has an upper limit...

Reply:
Agree, remarks should go to the very end - also given how it can be a substantial amount of text for BCL APIs.